.gitignore

```diff
@@ -1,5 +1,6 @@
 **/*/image.tar
 **/*/*.zip
+simulators/extras/export_data/
```

Comment:
- we should not exclude that folder (and delete its content in git) b/c it is part of the released zip file and used to import data during the PVTs/e2e tests in the CI/CD pipeline
- instead we should remove the content or the folder itself during the test execution